index.html

```diff
@@ -340,6 +340,139 @@ <h4 class="functionality__paragraph">Necesito Ayuda</h4>
+              <span class="most-popular__total">
+                <p class="most-popular__pric">$189.99</p>
+                <div class="most-popular__calificationd">
+                  <span class="most-popular__stard">&#9733;</span>
+                  <span class="most-popular__ratingd">5</span>
+                </div>
+              </span>
+            </div>
+          </div>
+          <div class="most-popular__cardddddd">
+            <a href="#" class="most-popular__link"></a>
+            <button class="most-popular__btton"> OFERTA</button>
+            <img class="most-popular__img" src="image/vacuum-cleaner.png" alt="">
+            <div class="most-popular__text">
+              <span class="most-popular__name">Microonda 0.8 de acero inoxidable</span>
+              <span class="most-popular__total">
+                <p class="most-popular__prices">$87.99 <s class="most-popular__number">$100.99</s>
+                </p>
+                <div class="most-popular__calification">
+                  <span class="most-popular__star">&#9733;</span>
+                  <span class="most-popular__rating">4</span>
+                </div>
+              </span>
+            </div>
+          </div>
+        </div>
+        <div class="most-popular__right">
+          <a href="#" class="most-popular__link"></a>
+          <div class="most-popular__right-clo">
+            <div class="most-popular__right-close">
+              <img class="most-popular__right-closee" src="image/Close.png" alt="Close">
+              <p class="Close">Envío a domicilio</p>
+            </div>
+            <div class="most-popular__right-check">
+              <img class="most-popular__right-checkk" src="image/Check.png" alt="Check">
+              <p class="Check">Retiro en tienda</p>
+            </div>
+          </div>
+          <div class="most-popular__rigth-calification">
+            <span class="most-popular__rigth-star">&#9733;</span>
+            <span class="most-popular__rigth-rating">3</span>
+          </div>
+          <img class="most-popular__right-img" src="image/Desktop-table.png" alt="">
+          <div class="most-popular__right-text">
+            <span class="most-popular__do">DO IT CENTER ONLINE </span>
+            <span class="most-popular__right-name">Desayunador con 2 bancas color espresso</span>
+            <div class="most-popular__right-total">
+              <p class="most-popular__right-price">209.99</p>
+              <div class="most-popular__rigiht-entrega">
+                <img class="most-popular__prioridad" src="image/Entregahoy.png" alt="pioridades"> Entrega hoy
+              </div>
+            </div>
+          </div>
+          <div class="most-popular__bord"></div>
+          <div class="most-popular__bord-botton">
+            <button class="most-popular__bord-button">VER PRODUCTO <img class="arrow" src="image/arrow-right.png" alt="">
+            </button>
```

Comment:
Pues si es 'ver producto' es mas logico que sea un link
```suggestion
            <a href="#" class="most-popular__bord-button">VER PRODUCTO <img class="arrow" src="image/arrow-right.png" alt=""></a>
```